fcli-core/fcli-aviator/build.gradle

```diff
@@ -0,0 +1,69 @@
+plugins {
+    id 'java'
+    id 'com.google.protobuf' version '0.9.4'
+}
+
+apply from: "${sharedGradleScriptsDir}/fcli-module.gradle"
+
+task zipResources_actions(type: Zip) {
+    destinationDirectory = file("${buildDir}/generated-zip-resources/com/fortify/cli/aviator")
+    archiveFileName = "actions.zip"
+    from("${projectDir}/src/main/resources/com/fortify/cli/aviator/actions/zip") {
+        // TODO We should also sign file; how do we invoke a sign operation from Gradle?
+        filter(line->project.version.startsWith('0.')
+                ? line
+                : line.replaceAll("https://fortify.github.io/fcli/schemas/action/fcli-action-schema-dev.json",
+                "https://fortify.github.io/fcli/schemas/action/fcli-action-schema-${fcliActionSchemaVersion}.json"))
+    }
+    include '*.yaml'
+}
+
+tasks.withType(JavaCompile) {
+    dependsOn 'generateProto'
+}
+
+dependencies {
+    implementation(project(":fcli-core:fcli-common"))
+
+    implementation("args4j:args4j:2.37")
```

Comment:
Probably best to add all these dependencies to fcli-bom, and remove the version numbers here? Haven you checked whether we can do without any of these dependencies, or possibly exclude some transitive dependencies? I think you already verified (and added a commit) to make sure that these dependencies operate correctly in GraalVM native binaries? Given the number of dependencies, does this significantly increase fcli jar/executable size?